scripts/src/logseq/tasks/malli.clj

```diff
@@ -18,3 +19,16 @@
       (println "Found errors:")
       (pprint/pprint errors))
     (println "Valid!")))
+
+(defn validate-global-config-edn
+  "Validate a global config.edn file"
+  [file]
+  (if-let [errors (->> file
+                       slurp
+                       edn/read-string
+                       (m/explain global-config-schema/Config-edn)
+                       me/humanize)]
```

Comment:
`seq` will be needed here, right?

Reply:
I would've thought too but the last two fns can return nil. I've removed the seq in the other place these fns are called